icecreamshop/src/main/java/com/summerschool/icecreamshop/model/Rate.java

```diff
@@ -0,0 +1,54 @@
+package com.summerschool.icecreamshop.model;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.*;
+
+@Entity
+@Table(name = "rates")
+public class Rate {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToMany(fetch = FetchType.LAZY)
+    @JoinColumn(name = "product_id")
+    private List<Product> product;
```

Comment:
Asta este o relatie de tipul One to Many in sensul - un produs poate avea mai multe rating-uri, si un rating poate fi pus pe un singur produs). Deci in clasa asta folosim Many to One cu fetch type lazy si avem referinta la un produs, iar in clasa Produs avem o lista de rates.